### ISSUE TITLE
Custom Dockerfile specified in docker-compose.yml is ignored

### ISSUE THREAD

**Issue by bladealslayer** [convox/rack#241]
`docker-compose.yml` allows you to specify custom Dockerfile for an image. It's useful if you'd like to build e.g. `web` and `worker` from same docker context (directory), but with different Dockerfiles. `convox` ignores that option though and builds both from default Dockerfile. This happens both when running locally with `convox start`, as well as when deployed to AWS. Example `docker-compose.yml`:

```
web:
  build: .
  command: bin/web
  environment:
    - DEVELOPMENT=true
    - MESSAGE
  ports:
    - 80:3000
    - 443:3000
  volumes:
    - .:/go/src/github.com/convox/demo
worker:
  build: .
  dockerfile: Dockerfile.worker
  command: bin/work
```

Using `docker-compose`, the worker gets properly built from `Dockerfile.worker`.

A workaround is to use different directory (and thus different Docker context) for the worker, using `build: my_worker_di`.

**Comment by nzoschke** [comment]
Thanks for the report. This directive should definitely be honored by `convox start` and `convox build`.

**Comment by nzoschke** [comment]
This was fixed in a5c817a7
